tasks/monitoring/systemd.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Create monitoring user
+  user:
+    name: bbb-exporter
+    shell: /usr/sbin/nologin
+    home: "{{ bbb_monitoring_systemd_directory }}"
+    create_home: false
+    system: true
+
+- name: Deploy git repository to host
+  git:
+    repo: https://github.com/greenstatic/bigbluebutton-exporter.git
+    dest: "{{ bbb_monitoring_systemd_directory }}"
+    version: "{{ bbb_monitoring_exporter_version }}"
+  notify:
+    - restart monitoring service
+
+- name: Change ownership directory
+  file:
+    path: "{{ bbb_monitoring_systemd_directory }}"
+    state: directory
+    owner: bbb-exporter
+    group: bbb-exporter
+    mode: '0755'
+
+- name: Ensure pip is installed
+  apt:
+    name:
+      - python3-pip
+      - python3-setuptools
+    state: "{{ bbb_state }}"
+
+- name: Install specified python requirements and custom Index URL
+  pip:
+    requirements: "{{ bbb_monitoring_systemd_directory }}/requirements.txt"
+  notify:
+    - restart monitoring service
+
+- name: Copy systemd service into place
+  copy:
+    src: "{{ bbb_monitoring_systemd_directory }}/extras/systemd/bigbluebutton-exporter.service"
+    dest: /lib/systemd/system/
+    owner: root
+    group: root
+    mode: '0644'
+    remote_src: true
+  notify:
+    - reload systemd
+    - restart monitoring service
+
+- name: Create /etc/bigbluebutton-exporter directory
+  file:
+    path: "/etc/bigbluebutton-exporter"
+    state: directory
+    mode: '0755'
+
+- name: Template exporter configuration
+  template:
+    src: "monitoring/bbb_exporter_secrets.env.j2"
+    dest: "/etc/bigbluebutton-exporter/settings.env"
+    mode: '0644'
+  notify:
+    - restart monitoring service
+
+- name: Copy monitoring nginx configuration file
+  template:
+    src: "monitoring/monitoring.nginx.j2"
+    dest: "/etc/bigbluebutton/nginx/monitoring.nginx"
+    mode: '0644'
+  notify:
+    - reload nginx
+
+- name: Generate htpasswd file for external access if enabled
+  htpasswd:
+    path: "/etc/bigbluebutton/nginx/.htpasswd"
+    name: "{{ bbb_monitoring_all_in_one_htpasswd_user }}"
+    password: "{{ bbb_monitoring_all_in_one_htpasswd }}"
+    owner: root
+    group: root
+    mode: 0644
+  when: bbb_monitoring_systemd_external
+  notify:
+    - reload nginx
+
+- name: start monitoring
+  systemd:
+    name: bigbluebutton-exporter
+    state: started
+    enabled: true
```

Comment:
In my opinion this is a bit verbose, since it duplicates many of the tasks from "all_in_one.yml". We might do a monitoring/common.yml @Davka @BrutalBirdie?

Reply:
@crosscodr there is now a main.yml file that handles the duplicate tasks